src/main.rs

```diff
@@ -12,15 +15,19 @@ use url::Url;
 mod checksum;
 use checksum::{Checksum, ChecksumValidator};
 
-static CHECKSUM_FILES: &[&str] = &[
-    "CHECKSUM.txt",
-    "checksum.txt",
-    "CHECKSUMS.txt",
-    "checksums.txt",
-    "SHASUMS256.txt",
-    "SHASUMS256",
-    // TODO add more patterns
-];
+static CHECKSUMS_FILES: Lazy<Vec<String>> = Lazy::new(|| {
+    vec![
+        "${path}/CHECKSUM.txt".to_string(),
```

Comment:
Add a comment directing to the help below  for explanations about `path` and `fullpath`?